lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -6893,7 +6987,14 @@ void EmitMethodFld(ParseNode *pnode, Js::RegSlot callObjLocation, Js::PropertyId
     bool isScoped = (byteCodeGenerator->GetFlags() & fscrEval) != 0 ||
         (isRoot && callObjLocation != ByteCodeGenerator::RootObjectRegister);
 
-    EmitMethodFld(isRoot, isScoped, pnode->location, callObjLocation, propertyId, byteCodeGenerator, funcInfo, registerCacheIdForCall);
```

Comment:
The stack isn't very big. Inside EmitCallTarget, we have a direct call to EmitMethodFld which is how we arrived here. Stack looks like this (hope formatting doesn't look terrible):

	Chakra.dll!EmitMethodFld(ParseNode * pnode, unsigned int callObjLocation, int propertyId, ByteCodeGenerator * byteCodeGenerator, FuncInfo * funcInfo, bool registerCacheIdForCall) Line 6992	C++
 	Chakra.dll!EmitCallTarget(ParseNode * pnodeTarget, int fSideEffectArgs, unsigned int * thisLocation, unsigned int * callObjLocation, ByteCodeGenerator * byteCodeGenerator, FuncInfo * funcInfo) Line 7256	C++
 	Chakra.dll!EmitCall(ParseNode * pnode, unsigned int rhsLocation, ByteCodeGenerator * byteCodeGenerator, FuncInfo * funcInfo, int fReturnValue, int fEvaluateComponents, int fHasNewTarget, unsigned int overrideThisLocation) Line 7573	C++
 	Chakra.dll!Emit(ParseNode * pnode, ByteCodeGenerator * byteCodeGenerator, FuncInfo * funcInfo, int fReturnValue, bool isConstructorCall, ParseNode * bindPnode) Line 9881	C++
 	Chakra.dll!ByteCodeGenerator::EmitTopLevelStatement(ParseNode * stmt, FuncInfo * funcInfo, int fReturnValue) Line 956	C++